static/js/containers/pages/b2b/B2BPurchasePage.js

```diff
@@ -126,9 +126,15 @@ export class B2BPurchasePage extends React.Component<Props, State> {
         ) : (
           <B2BPurchaseForm
             onSubmit={this.onSubmit}
-            products={products.filter(
-              product => product.visible_in_bulk_form === true
-            )}
+            products={products
```

Comment:
Please add this sorting capability in the API view instead of the front end here. It's a little bit tricky since the field we're sorting on is part of a generic relation. There isn't a good way to apply the sort to the queryset, so I think we should just override the `list` method of the view to sort the results after they're serialized and return a new response with sorted items. Steps should be something like this:

1. Add `def list(self, request, *args, **kwargs):` to `ecommerce.views.ProductViewSet`
1. Call `super().list` in that method, which returns a `rest_framework.response.Response` object. 
1. If that response status is ok (2xx), and the user's request URL included `sort=title` in the querystring params, get the data from the response, sort the items by title, and update the `data` attribute of that response object with the sorted list.

If there's an issue with any of that, let me know

Reply:
Updated